<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>550</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_check">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>출결</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_message">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>메시지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cal">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>일정</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_login2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>184</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="btn_in">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>입실</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_out">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>퇴실</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_go">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>외출</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_logout">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>로그아웃</string>
     </property>
    </widget>
    <widget class="QLabel" name="attendance">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>30</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>출석</string>
     </property>
    </widget>
    <widget class="QLabel" name="late">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>450</y>
       <width>30</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>지각</string>
     </property>
    </widget>
    <widget class="QLabel" name="leave">
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>450</y>
       <width>30</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>조퇴</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home2">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>복귀</string>
     </property>
    </widget>
    <widget class="QLabel" name="goingout">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>450</y>
       <width>30</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>외출</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="checkin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="checkout">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="checkgo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="checkback">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>310</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="absent">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>450</y>
       <width>30</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>결석</string>
     </property>
    </widget>
    <widget class="QLabel" name="attendancecount">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>420</y>
       <width>20</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>46</string>
     </property>
    </widget>
    <widget class="QLabel" name="latecount">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>420</y>
       <width>20</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="leavecount">
     <property name="geometry">
      <rect>
       <x>217</x>
       <y>420</y>
       <width>20</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="goingoutcount">
     <property name="geometry">
      <rect>
       <x>283</x>
       <y>420</y>
       <width>20</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="absentcount">
     <property name="geometry">
      <rect>
       <x>348</x>
       <y>420</y>
       <width>20</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="okbutton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QPushButton" name="btn_tlogout">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>로그아웃</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>71</y>
       <width>411</width>
       <height>461</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>이름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>입실시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>외출시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>복귀시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>퇴실시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>출석횟수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>지각횟수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>조퇴횟수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>외출횟수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>결석횟수</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="okbutton2">
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>출결 확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>421</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>351</x>
       <y>351</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>일정 추가</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="scheduleadd">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>352</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="schedulespace">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>411</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLineEdit" name="message">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>475</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_send">
     <property name="geometry">
      <rect>
       <x>339</x>
       <y>475</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>전송</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
    <widget class="QListWidget" name="messagespace">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>380</width>
       <height>380</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="teacherlist">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>이상복</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>류홍걸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>조동현</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QPushButton" name="btn_home6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
    <widget class="QListWidget" name="messagespace2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>380</width>
       <height>380</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="studentlist">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>강민영</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>고연재</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>김기태</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>김명은</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>김성일</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>김연수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>노도현</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>류가미</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>박규환</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>박성빈</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>박시형</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>박의용</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>오송화</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이범규</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이보라</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이소윤</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이여름</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이지혜</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이현도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>임성경</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>임영효</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>임홍선</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>장은희</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>정연우</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>정철우</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>주민석</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>최지혁</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
